internal/service/server/block_storage_test.go

```diff
@@ -268,6 +263,34 @@ func testAccCheckBlockStorageExistsWithProvider(n string, i *server.BlockStorage
 	}
 }
 
+func testAccCheckClassicBlockStorageExistsWithProvider(n string, i *server.BlockStorage, provider *schema.Provider) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		rs, ok := s.RootModule().Resources[n]
+		if !ok {
+			return fmt.Errorf("not found: %s", n)
+		}
+
+		if rs.Primary.ID == "" {
+			return fmt.Errorf("no ID is set")
+		}
+
+		time.Sleep(time.Second * 10)
```

Comment:
Adding sleep is usually not a good approach. It feels like the server's APIs for BlockStorage don't match the resource, which is something that needs to be handled through the API.

Reply:
I agree that adding sleep is not appropriate. I have requested an improvement from the server maintainer. However, we are in the same situation as in case https://github.com/NaverCloudPlatform/terraform-provider-ncloud/pull/368, we don't know when the API will be improved. So, I added a temporary code to pass the AccTest until it's improved.

Reply:
Yes, I understand that situation, and I made a bad precedent. The source pipeline case was for an OSSCA project with a limited time, and it assumes that if the API is fixed we should modify it again. So I think we don't need to rush things here too.

Reply:
I understand, I will fix it after improving the server API.